presto-main/src/main/java/io/prestosql/sql/planner/iterative/rule/NormalizeOrExpressions.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.sql.planner.iterative.rule;
+
+import io.prestosql.metadata.Metadata;
+
+import static io.prestosql.sql.planner.OrExpressionNormalizer.rewrite;
+
+public class NormalizeOrExpressions
```

Comment:
Instead of creating this class maybe you could introduce a factory method `ExpressionRewriteRuleSet.of(OrExpressionNormalizer::rewrite)`?

Reply:
ping